src/util/util_string.cpp

```diff
@@ -35,4 +35,9 @@ namespace dxvk::str {
         auto w = std::wstring(reinterpret_cast<wchar_t*>(nvus));
         return {w.begin(), w.end()};
     }
+
+    void tonvss(NvAPI_ShortString nvss, std::string str) {
+        str.resize(NVAPI_SHORT_STRING_MAX);
```

Comment:
shouldn't this be `NVAPI_SHORT_STRING_MAX-1` so we have space for the terminating NUL?

```c++
failcraft ivyl ~ % cat test.cpp
#include <iostream>
#include <string>

int main()
{
	std::string str("1234567890");
	str.resize(2);
	std::cout << str.c_str() << std::endl;
	return 0;
}
failcraft ivyl ~ % g++ test.cpp -o test && ./test
12
```